src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -400,21 +405,28 @@ ExitInfo ExecutorWorker::handleCreateOp(SyncOpPtr syncOp, std::shared_ptr<Abstra
 
         if (job && syncOp->affectedNode()->type() == NodeType::Directory) {
             // Propagate the directory creation immediately in order to avoid blocking other dependant job creation
-            if (ExitInfo exitInfo = runCreateDirJob(syncOp, job); !exitInfo) {
```

Comment:
Maybe the logic of `runCreateDirJob` could include this back error check. 
This is just a tough, here is a dedicated Jira card for a dedicated PR->
https://infomaniak.atlassian.net/jira/software/c/projects/KDESKTOP/boards/341